crates/sui-json-rpc-types/src/lib.rs

```diff
@@ -51,6 +56,209 @@ use sui_types::sui_serde::{Base64, Encoding};
 mod rpc_types_tests;
 
 pub type GatewayTxSeqNumber = u64;
+pub type SuiMoveTypeParameterIndex = u16;
+#[derive(Serialize, Deserialize, Debug, JsonSchema)]
+pub struct SuiMoveAbilitySet(u8);
+
+#[derive(Serialize, Deserialize, Debug, JsonSchema)]
+pub enum SuiMoveVisibility {
+    Private = 0x0,
+    Public = 0x1,
+    Friend = 0x3,
```

Comment:
```suggestion
    Private,
    Public,
    Friend,
```

nit: We shouldn't need to define the value as they should be serialized to strings